### ISSUE TITLE
Incorrect indentation of CSS inside SVG

### ISSUE THREAD

**Issue by trevorparscal** [ota-meshi/eslint-plugin-svelte#405]
### Before You File a Bug Report Please Confirm You Have Done The Following...

- [X] I have tried restarting my IDE and the issue persists.
- [X] I have updated to the latest version of the packages.

### What version of ESLint are you using?

8.36.0

### What version of `eslint-plugin-svelte` are you using?

2.21.0

### What did you do?

Using no special configuration beyond extending from `plugin:svelte/recommended`, when I add a style tag to an SVG, the CSS properties are not indented inside each rule.

```svelte
<svg>
  <style>
    .test {
      fill: red;
    }
  </style>
</svg>
```

The `svelte/indent` rule shows a fixable error, "Expected indentation of 4 spaces but found 6 spaces."

### What did you expect to happen?

```svelte
<svg>
  <style>
    .test {
      fill: red;
    }
  </style>
</svg>
```

### What actually happened?

```svelte
<svg>
  <style>
    .test {
    fill: red;
    }
  </style>
</svg>
```

### Link to **GitHub Repo** with Minimal Reproducible Example

https://ota-meshi.github.io/eslint-plugin-svelte/playground/#eJyrVkrOT0lVslKyKS5Lt4vJU1CwKS6pzEkFMxUU9EpSi0sUqiEcBYW0zJwcK4Wi1BRriEgtWIM+TAeQBTREqRYAXrgaFw==

### Additional comments

I would expect this is a bug, but if it's intended, perhaps there could be a way to adjust this behavior in the `svelte/indent` rule's options.

**Comment by ota-meshi** [comment]
Thank you for this report.

I think it's a bug in the `svelte/indent` rule.
